recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class MsdfAtlasGenConan(ConanFile):
+    name = "msdf-atlas-gen"
+    license = "MIT"
+    homepage = "https://github.com/Chlumsky/msdf-atlas-gen"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "MSDF font atlas generator"
+    topics = ("conan", "msdf-atlas-gen", "msdf", "font", "atlas")
```

Comment:
Adding `conan` to topics is superfluous.
```suggestion
    topics = ("msdf-atlas-gen", "msdf", "font", "atlas")
```